scripts/yocto-build-and-test.sh

```diff
@@ -499,6 +499,7 @@ build_and_test_client() {
         if is_building_extra_images_for_board "$board_name" \
                 && [[ $image_name == core-image-full-cmdline ]]; then
             images_to_build+=" mender-image-full-cmdline-rofs"
+            images_to_build+=" mender-image-full-cmdline-rofs-commercial"
```

Comment:
You need to check if the recipe exists, as we do for the ones below. So something like:
```
           if [[ -f $WORKSPACE/meta-mender-demo/recipes-extended/images/mender-image-full-cmdline-rofs-commercial.bb ]]; then
                images_to_build+=" mender-image-full-cmdline-rofs-commercial"
            fi
```